### PR TITLE
feat: develop Tetragon Enterprise page

### PR DESCRIPTION
**Pull request by annaindistress**
**Describe what changes this pull request brings**

- Add Enterprise page
- Update `Welcome to the community!` section

**Steps to check:**
1. Open [preview](https://deploy-preview-3103--tetragon.netlify.app/enterprise/)
2. Make sure that everything looks and works as expected

### CODE REVIEW

**Review by yasell — APPROVED** [review]

File: docs/content/en/enterprise.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+					rel="noopener noreferrer">
+					<img src="/svgs/social/linkedin-icon.svg" width="64" height="64" loading="lazy" alt="" />
+					<span>Follow us on&nbsp;LinkedIn</span>
+				</a>
+			</li>
+			<li class="social__item social__item--blue">
+				<a class="social__link" href="https://isovalent.com/labs/tetragon-getting-started/" target="_blank"
+					rel="noopener noreferrer">
+					<img src="/svgs/social/lab-icon.svg" width="64" height="64" loading="lazy" alt="" />
+					<span>Explore Tetragon&nbsp;lab</span>
```
Comment (xmulligan) [inline docs/content/en/enterprise.html:106]:
```suggestion
					<span>Explore Tetragon&nbsp;labs</span>
```

Comment (annaindistress) [reply docs/content/en/enterprise.html]: Fixed

**Review by mtardy — COMMENTED** [review]
Technically looks good, not sure repeating the community thing in the enterprise is important but if it's what was decided I have no strong feeling.

**Review by mtardy — APPROVED** [review]

**Review by michi-covalent — APPROVED** [review]
let's wait for an ack from @xmulligan

**Review by xmulligan — APPROVED** [review]

File: docs/content/en/_index.html [hunk 2]
```diff
@@ -557,95 +557,44 @@ <h2 class="title">Event in a box</h2>
...
-                                <path d="M7.11108 1L12 5.7735L7.11108 10.7444" stroke="currentColor" stroke-width="1.5"
-                                    stroke-linecap="round" stroke-linejoin="round" />
-                            </svg>
-                        </span>
-                    </div>
-                </a>
-            </li>
-        </ul>
-    </div>
+		<h2 class="title social__title">Welcome to the community!</h2>
```
[the remark this answers is not in the document]
Comment (annaindistress) [reply docs/content/en/_index.html]: Done - used a shortcode for the social section

File: docs/content/en/enterprise.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	<div class="hero hero--inner">
+		<div class="container">
+			<div class="hero__intro">
+				<h1 class="title title--xl">
+					Tetragon Enterprise Distributions & Training
+				</h1>
+				<p>
+					Initially created by Isovalent, Tetragon is a CNCF-backed project dedicated to real-time
+					observability and security for Kubernetes. Certified partners provide enterprise distributions,
+					training, and&nbsp;commercial support for Tetragon.
```
Comment (kkourt) [inline docs/content/en/enterprise.html:19]: Can we change "Certified partners provide enterprise distributions, training, and commercial support for Tetragon" to "The listed partners offer enterprise distributions, training, and commercial support for Tetragon", which is the wording that Cilium uses (https://cilium.io/enterprise/) ?

Comment (annaindistress) [reply docs/content/en/enterprise.html]: Done ✅

**Review by kkourt — APPROVED** [review]
Thanks!

**Comment by mtardy** [comment]
All right thanks everyone, and special thanks to @annaindistress ! :)

**Pull request merged**